[CI] test loss with kwargs is broken on nightly

## 🐛 Bug description

```
    def test_kwargs_loss():
        loss = Loss(nll_loss)
    
        y_pred, y, _ = y_test_1()
        loss.update((y_pred, y, {"weight": torch.tensor([0, 0, 0], dtype=torch.float)}))
>       assert_almost_equal(loss.compute(), 0)
E       AssertionError: 
E       Arrays are not almost equal to 7 decimals
E        ACTUAL: nan
E        DESIRED: 0
```

- https://github.com/pytorch/ignite/pull/2236/checks?check_run_id=3782539427#step:15:1375


## Environment

- PyTorch Version (e.g., 1.4):
- Ignite Version (e.g., 0.3.0):
- OS (e.g., Linux):
- How you installed Ignite (`conda`, `pip`, source):
- Python version:
- Any other relevant information:

--- a/tests/ignite/metrics/test_loss.py
+++ b/tests/ignite/metrics/test_loss.py
@@ -138,8 +138,10 @@ def test_kwargs_loss():
     loss = Loss(nll_loss)
 
     y_pred, y, _ = y_test_1()
-    loss.update((y_pred, y, {"weight": torch.tensor([0, 0, 0], dtype=torch.float)}))
-    assert_almost_equal(loss.compute(), 0)
+    kwargs = {"weight": torch.tensor([0.1, 0.1, 0.1])}
+    loss.update((y_pred, y, kwargs))
+    expected_value = nll_loss(y_pred, y, **kwargs)
+    assert_almost_equal(loss.compute(), expected_value)
 
 
 def test_reset():

